etl/models/models.go

```diff
@@ -16,15 +16,28 @@ import (
 )
 
 type RecordType string
+type StatesType string
+type RequestsType string
 
 func RecordTypeFromTypes(rt types.RecordType) RecordType {
 	return []RecordType{"state", "request", "result"}[rt]
 }
 
 const (
-	State   RecordType = "state"
-	Request RecordType = "request"
-	Result  RecordType = "result"
+	StateType   RecordType = "state"
+	RequestType RecordType = "request"
+	ResultType  RecordType = "result"
+)
+
+const (
+	Activate   StatesType = "activate"
+	Amend      StatesType = "amend"
+	Deactivate StatesType = "deactivate"
+)
+
+const (
+	Incoming RecordType = "incoming"
```

Comment:
`Incoming` is `RequestsType`, not `RecordType`
`Outgoing` too

Reply:
Done